pallets/identity/src/benchmarking.rs

```diff
@@ -183,108 +269,114 @@ benchmarks! {
         )?;
 
         let owner_auth_id =  Module::<T>::add_auth(
-            target_did,
-            Signatory::Account(new_key),
-            AuthorizationData::RotatePrimaryKey(target_did),
+            target.did(), signatory,
+            AuthorizationData::RotatePrimaryKey(target.did()),
             None,
         );
-    }: _(new_key_origin, owner_auth_id, Some(cdd_auth_id))
+    }: _(new_key.origin, owner_auth_id, Some(cdd_auth_id))
 
     change_cdd_requirement_for_mk_rotation {}: _(RawOrigin::Root, true)
 
     join_identity_as_key {
-        let (_, _, target_did) = make_account::<T>("target", SEED);
-        let (new_key, new_key_origin) = make_account_without_did::<T>("key", SEED);
+        let target = User::<T>::new("target", SEED);
+        let new_key = User::<T>::without_did("key", SEED);
```

Comment:
The variable name is obsolete. Perhaps `new_user` or maybe something more informative.